[2025-04-18T16:46:34.463+1000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-18T16:46:34.513+1000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: executing_multiple_tasks.taskB manual__2025-04-18T06:46:09.446866+00:00 [queued]>
[2025-04-18T16:46:34.540+1000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: executing_multiple_tasks.taskB manual__2025-04-18T06:46:09.446866+00:00 [queued]>
[2025-04-18T16:46:34.541+1000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-18T16:46:34.581+1000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): taskB> on 2025-04-18 06:46:09.446866+00:00
[2025-04-18T16:46:34.594+1000] {standard_task_runner.py:72} INFO - Started process 85280 to run task
[2025-04-18T16:46:34.617+1000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'executing_multiple_tasks', 'taskB', 'manual__2025-04-18T06:46:09.446866+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/execute_multiple_tasks.py', '--cfg-path', '/var/folders/05/dgx_ky6d3n9f_vxn26lcln1h0000gn/T/tmpc8wxxlz3']
[2025-04-18T16:46:34.630+1000] {standard_task_runner.py:105} INFO - Job 17: Subtask taskB
[2025-04-18T16:46:34.775+1000] {task_command.py:467} INFO - Running <TaskInstance: executing_multiple_tasks.taskB manual__2025-04-18T06:46:09.446866+00:00 [running]> on host anurags-macbook-pro.local
[2025-04-18T16:46:35.004+1000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='anuragthakur' AIRFLOW_CTX_DAG_ID='executing_multiple_tasks' AIRFLOW_CTX_TASK_ID='taskB' AIRFLOW_CTX_EXECUTION_DATE='2025-04-18T06:46:09.446866+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-18T06:46:09.446866+00:00'
[2025-04-18T16:46:35.007+1000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-18T16:46:35.069+1000] {subprocess.py:78} INFO - Tmp dir root location: /var/folders/05/dgx_ky6d3n9f_vxn26lcln1h0000gn/T
[2025-04-18T16:46:35.072+1000] {subprocess.py:88} INFO - Running command: ['/bin/bash', '-c', '\n        echo TASK B has started!\n        sleep 4\n        echo TASK B has ended!\n    ']
[2025-04-18T16:46:35.092+1000] {subprocess.py:99} INFO - Output:
[2025-04-18T16:46:35.106+1000] {subprocess.py:106} INFO - TASK B has started!
[2025-04-18T16:46:39.134+1000] {subprocess.py:106} INFO - TASK B has ended!
[2025-04-18T16:46:39.136+1000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-04-18T16:46:39.196+1000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-18T16:46:39.201+1000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=executing_multiple_tasks, task_id=taskB, run_id=manual__2025-04-18T06:46:09.446866+00:00, execution_date=20250418T064609, start_date=20250418T064634, end_date=20250418T064639
[2025-04-18T16:46:39.268+1000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-18T16:46:39.345+1000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-18T16:46:39.347+1000] {local_task_job_runner.py:245} INFO - ::endgroup::
